Split/Models/Key.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 
 public class Key: NSObject {
 
-    let matchingKey: String
-    let bucketingKey: String?
+    public let matchingKey: String
+    public let bucketingKey: String?
```

Comment:
Double checking if this is on purpose

Reply:
Yup, I'll need public access to those properties.
Thanks for the review!